rakelib/form526.rake

```diff
@@ -518,11 +518,15 @@ namespace :form526 do
         end
 
         vname = "#{fs.auth_headers['va_eauth_firstName']} #{fs.auth_headers['va_eauth_lastName']}"
-        icn = Account.lookup_by_user_uuid(fs.user_uuid).first&.icn
+        icn = fs.user_account&.icn
```

Comment:
I dont know of anyone using these rake things anymore, good to update them, but I am not reviewing this part very hard just fyi